Author: take of ciwiki-1.7.5
Maintainer: Artjom Slepnjov "Shellgen"
Description: fix - work in blink engine (google) web-browsers
Ver: didiwiki-0.8
Date: 2025-03-14 09:00 UTC
==============================================================
--- a/src/http.c
+++ b/src/http.c
@@ -442,8 +442,6 @@
       int n_bytes = res->data_len;
       
       /* Dont send '\0' in response */
-      if (res->data[n_bytes-1] == '\0') 
-	n_bytes--;
 
       fwrite(res->data, 1, n_bytes, stdout);
     }
